go.mod

```diff
@@ -265,14 +266,14 @@ require (
 	nhooyr.io/websocket v1.8.11 // indirect
 	pgregory.net/rapid v1.2.0 // indirect
 	rsc.io/qr v0.2.0 // indirect
-	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
 replace (
 	// use cosmos fork of keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
 	// use Cosmos geth fork
-	github.com/ethereum/go-ethereum => github.com/cosmos/go-ethereum v1.15.11-cosmos-0
+	// branch: cosmos-1-16
+	github.com/ethereum/go-ethereum => github.com/yihuang/go-ethereum v1.10.21-0.20250721081216-98795aa589d5
```

Comment:
Is there a reason for this change? I checked github.com/yihuang/go-ethereum v1.10.21-0.20250721081216-98795aa589d5 and found that it is in the same state as github.com/cosmos/go-ethereum v1.15.11-cosmos-0.

Reply:
reverted this dependency change.